dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -442,13 +442,13 @@ def setUpClass(cls):
         cls.trained_schema = dp.Profiler(test_dataset, len(test_dataset))
 
     def test_correct_rows_ingested(self):
-        self.assertEqual(['', 'nan', 'None', 'null'],
+        self.assertCountEqual(['', 'nan', 'None', 'null'],
```

Comment:
Checks that the elements are the same, but not the order

https://docs.python.org/3.2/library/unittest.html#unittest.TestCase.assertCountEqual

Reply:
can we convert to `set` then, instead of assert count?

Reply:
assert count is the appropriate mechanism, why do we need to change the type as opposed to using the correct assert?

Reply:
kk, I thought it asserts only the count, not the elements